tests/test_app/TestApp/Controller/ArticlesController.php

```diff
@@ -23,5 +23,5 @@ class ArticlesController extends Controller
     /**
      * @var string
      */
-    protected ?string $modelClass = ArticlesTable::class;
+    protected ?string $defaultTable = ArticlesTable::class;
```

Comment:
I bet there are plenty of applications that are using modelClass. We could emit a deprecation for this during Controller::__construct().